src/api/errors/levels/BaseError.ts

```diff
@@ -0,0 +1,27 @@
+class BaseError extends Error {
+  type: string
+  code: number
+  title: string
+  level: string
+
+  constructor (code: number, title: string, message: string, level: string){
+    super(message)
+    this.type = 'BaseError'
+    this.code = code
+    this.title = title
+    this.level = level
+  }
+
+  toJSON () {
```

Comment:
Parentheses must be attached to the method name.
```suggestion
  toJSON() {
```